adapters/repos/db/lsmkv/ent/doc.go

```diff
@@ -0,0 +1,2 @@
+// ent contains common types used througout various lsmkv (sub-)packages
+package ent
```

Comment:
what does `ent` stands for?

Reply:
For `entities`. The idea was similar to the `entities` that we have at root level, these are the inner-most structures that all other packages can depend on (and they depend on nothing). But not in the global scope, but just within the `lmskv` scope. 

Since you had to ask, `entities` would probably be a better name?.

Reply:
ok, got it 👍 
I'm for `entities` then.